tests/tracking/fluent/test_fluent.py

```diff
@@ -423,6 +424,60 @@ def test_search_experiments(tmp_path):
     assert [e.name for e in experiments] == sorted(active_experiment_names, reverse=True)[:3]
 
 
+def test_search_registered_models(tmp_path):
+    sqlite_uri = "sqlite:///{}".format(tmp_path.joinpath("test.db"))
+    mlflow.set_tracking_uri(sqlite_uri)
+
+    num_all_models = SEARCH_REGISTERED_MODEL_MAX_RESULTS_DEFAULT + 1
+    num_azure_models = num_all_models // 4
+    num_boston_models = num_all_models - num_azure_models
+
+    azure_model_names = [f"AzureWeatherForecastModel_{i}" for i in range(num_azure_models)]
+    boston_model_names = [f"BostonWeatherForecastModel_{i}" for i in range(num_boston_models)]
+    model_names = boston_model_names + azure_model_names
+
+    tag_values = ["x", "x", "y"]
+    for (idx, model_name) in enumerate(model_names):
+        tags = None
+        if idx < len(tag_values):
+            tags = {"tag": tag_values[idx]}
+
+        MlflowClient().create_registered_model(model_name, tags=tags)
```

Comment:
```suggestion
    tag_values = ["x", "x", "y"]
    for (tag, model_name) in zip_longest(model_names, tag_values):
        MlflowClient().create_registered_model(model_name, tags={"tag": tag})
```

We can use `zip_longest` here.